packages/block-editor/src/components/inserter/menu.js

```diff
@@ -67,6 +67,8 @@ function InserterMenu(
 	const [ patternFilter, setPatternFilter ] = useState( 'all' );
 	const [ selectedMediaCategory, setSelectedMediaCategory ] =
 		useState( null );
+	const isLargeViewport = ! useViewportMatch( 'medium', '<' );
```

Comment:
When the viewport width is between `783px` and `959px`, the zoom out toggle button is not displayed, but when we select the Patterns tab, zoom out is enabled. When the zoom out toggle button is not displayed, I think the zoom out mode itself should not be enabled. What do you think?

Reply:
Good catch 👍🏻 Updated! Now it should only enter zoom out for the pattern inserter when the zoom out button in the header is showing.